pyproject.toml

```diff
@@ -66,9 +67,29 @@ furo = "^2023.9.10"
 mock = "^5.1.0"
 autodoc = "^0.5.0"
 
+[tool.poetry-dynamic-versioning]
+enable = true
+style = "pep440"
+vcs = "git"
+
+[tool.poetry-dynamic-versioning.substitution]
+files = ["pycytominer/__about__.py"]
+
+[tool.commitizen]
+# This version is used for changelog tracking and is updated using `cz bump`
```

Comment:
Would it make sense to add notes to the contributing docs about how and when to use `cz bump`?

Reply:
I had this idea too, but figured it would be better to write something describing the full release process. This section will be easier to do once we actually complete a release. :) I've created an issue to track this docs request though. #343